addons/website_forum/populate/forum_post.py

```diff
@@ -17,6 +17,12 @@ class Post(models.Model):
 
     def _populate_factories(self):
         forum_ids = self.env.registry.populated_models['forum.forum']
+        tag_ids = {
+            tag[0].id: tag[1] for tag in self.env["forum.tag"]._read_group(
+                [('forum_id', 'in', forum_ids)],
+                ['forum_id'],
+                ['id:array_agg'],
+        )}
         hours = [_ for _ in range(1, 49)]
```

Comment:
while we change the file
```suggestion
        hours = list(range(1, 49))
```